src/generic/stage1/json_string_scanner.h

```diff
@@ -125,12 +133,14 @@ simdjson_really_inline json_string_block json_string_scanner::next(const simd::s
   prev_in_string = uint64_t(static_cast<int64_t>(in_string) >> 63);
 
   // Use ^ to turn the beginning quote off, and the end quote on.
-  return {
+
+  // Should be passed as a move.
```

Comment:
The comment here probably needs to say what should be passed as a move (or by whom), since there are no moves :)

Reply:
(i.e. I was confused reading it)

Reply:
The new comment:
```C++
  // We are returning a function-local object so either we get a move construction
  // or we get copy elision.
```

Reply:
Oh. Move constructor (typo)